pythonforandroid/prerequisites.py

```diff
@@ -216,13 +247,123 @@ def darwin_installer(self):
         os.environ["JAVA_HOME"] = jdk_path
 
 
-def check_and_install_default_prerequisites():
-    DEFAULT_PREREQUISITES = dict(darwin=[JDKPrerequisite()], linux=[], all_platforms=[])
+class OpenSSLPrerequisite(Prerequisite):
+    name = "openssl@1.1"
+    mandatory = dict(linux=False, darwin=True)
+    installer_is_supported = dict(linux=False, darwin=True)
+
+    def darwin_checker(self):
+        if self._darwin_get_brew_formula_location_prefix("openssl@1.1", installed=True):
+            return True
+        else:
+            return False
```

Comment:
Why don't we inline that one too here and in the other places?
```suggestion
        return self._darwin_get_brew_formula_location_prefix("openssl@1.1", installed=True)
```